lib/delayed/performable_method.rb

```diff
@@ -22,22 +23,54 @@ def display_name
       end
     end
 
+    def kwargs
```

Comment:
**Question:** Will this be overridden by the `kwargs` attr_accessor? If it is, we could have an `attr_writer` for kwargs, and use this as the reader.

Reply:
kwargs method should override given it's defined later , but it makes sense to use attr_writer with the method above to avoid ambiguity. Fixed!